resources/user-guides/tutorial-1-edge-extension-setup.md

```diff
@@ -52,7 +52,6 @@ To create a configuration identifier use the following steps:
 
 1. To send events to Adobe Experience Platform, enable the `Adobe Experience Platform` section as shown below:
    * Select `Prod` for `AEP Sandbox`.
```

Comment:
Can you please include a link to more info about sandboxes here?
* Select `Prod` for `AEP Sandbox`. For more details about Sandboxes, see [Sandboxes overview](https://experienceleague.adobe.com/docs/experience-platform/sandbox/home.html?lang=en)

Reply:
Done!